configuration/reports/reportdescriptors/dataexports/sql/labOrdersReport.sql

```diff
@@ -131,10 +131,25 @@ UPDATE temp_report t SET status =
 UPDATE temp_report t SET orderer = PROVIDER(t.order_encounter_id);
 UPDATE temp_report t SET orderer_provider_type = PROVIDER_TYPE(t.order_encounter_id);
 UPDATE temp_report t SET ordering_location = ENCOUNTER_LOCATION_NAME(t.order_encounter_id);
-UPDATE temp_report t SET specimen_collection_datetime = ENCOUNTER_DATE(t.order_encounter_id);
-UPDATE temp_report t SET test_location = OBS_VALUE_CODED_LIST(t.specimen_encounter_id, 'PIH','11791',@locale);
-UPDATE temp_report t SET result_date = OBS_VALUE_DATETIME(t.specimen_encounter_id,'PIH','Date of test results'); 
-UPDATE temp_report t SET collection_date_estimated = OBS_VALUE_CODED_LIST(t.specimen_encounter_id, 'PIH','11781',@locale);
+
+update temp_report t
+  inner join encounter e on e.encounter_id = t.specimen_encounter_id
```

Comment:
This is a different column that the old version.  Previously you were getting the date of the order encounter.  Here you are getting the date of the specimen encounter (which seems right - was this a bug fix?)